### PR TITLE
update elrond-go-core

### PR DESCRIPTION
**Pull request by miiu96**
- Updated elrond-go-core
- Removed duplicated structures

### CODE REVIEW

File: data/transaction.go [hunk 1]
```diff
@@ -27,61 +27,9 @@ type Transaction struct {
...
-	NotarizedAtDestinationInMetaNonce uint64                                `json:"notarizedAtDestinationInMetaNonce,omitempty"`
-	NotarizedAtDestinationInMetaHash  string                                `json:"notarizedAtDestinationInMetaHash,omitempty"`
-	MiniBlockType                     string                                `json:"miniblockType,omitempty"`
-	MiniBlockHash                     string                                `json:"miniblockHash,omitempty"`
-	Timestamp                         int64                                 `json:"timestamp,omitempty"`
-	Status                            transaction.TxStatus                  `json:"status,omitempty"`
-	HyperblockNonce                   uint64                                `json:"hyperblockNonce,omitempty"`
-	HyperblockHash                    string                                `json:"hyperblockHash,omitempty"`
-	Receipt                           *transaction.ApiReceipt               `json:"receipt,omitempty"`
-	ScResults                         []*transaction.ApiSmartContractResult `json:"smartContractResults,omitempty"`
```
Comment (andreibancioiu) [inline data/transaction.go:69]: So, no JSON field had changed, right?

**Review by andreibancioiu — COMMENTED** [review]
Just a question (about not modifying any field names).

File: data/transaction.go (hunk 1 above)
Comment (miiu96) [reply data/transaction.go:69]: Yes are the same 
https://github.com/ElrondNetwork/elrond-go-core/blob/ebce3311fa4cb0661b8347e88e74b4bcdcd93afe/data/transaction/apiTransactionResult.go#L50

File: data/block.go [hunk 2]
```diff
@@ -91,33 +51,22 @@ type HyperblockApiResponsePayload struct {
...
-	PrevEpochStartHash               string `json:"prevEpochStartHash"`
+	Hash                   string                              `json:"hash"`
+	PrevBlockHash          string                              `json:"prevBlockHash"`
+	Nonce                  uint64                              `json:"nonce"`
+	Round                  uint64                              `json:"round"`
+	Epoch                  uint32                              `json:"epoch"`
+	NumTxs                 uint32                              `json:"numTxs"`
+	AccumulatedFees        string                              `json:"accumulatedFees,omitempty"`
+	DeveloperFees          string                              `json:"developerFees,omitempty"`
+	AccumulatedFeesInEpoch string                              `json:"accumulatedFeesInEpoch,omitempty"`
```
Comment (bogdan-rosianu) [inline data/block.go:62]: please revert: don't change the json tags!! make sure the new tags are exactly the same as before

File: process/hyperblockBuilder_test.go [hunk 3]
```diff
@@ -1,48 +1,49 @@
 package process
 
 import (
+	"github.com/ElrondNetwork/elrond-go-core/data/api"
```
Comment (bogdan-rosianu) [inline process/hyperblockBuilder_test.go:4]: imports not sorted

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: data/block.go (hunk 2 above)
Comment (miiu96) [reply data/block.go:62]: structure is the same

**Comment by miiu96** [comment]
image

File: process/hyperblockBuilder_test.go (hunk 3 above)
Comment (miiu96) [reply process/hyperblockBuilder_test.go]: done.

**Review by andreibancioiu — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by andreibancioiu — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Review by popenta — APPROVED** [review]
system test passed

**Review by andreibancioiu — APPROVED** [review]

**Pull request merged**